cypress/integration/scriptSpec.js

```diff
@@ -0,0 +1,56 @@
+import config from '../support/config';
+
+[
+  { service: 'news', url: `/news/articles/${config.assets.news}` },
+  {
+    service: 'persian',
+    url: `/persian/articles/${config.assets.persian}`,
+  },
+  { service: 'igbo', url: `/igbo` },
+  { service: 'yoruba', url: `/yoruba` },
+  { service: 'pidgin', url: `/pidgin` },
+].forEach(({ service, url }) => {
+  describe(`Script src - ${service}`, () => {
+    beforeEach(() => {
+      cy.visit(url);
+    });
+
+    it('should only have expected bundle script tags', () => {
+      cy.onLocal(() => {
+        cy.get('script[src]').each($p =>
+          expect($p.attr('src')).to.match(
+            new RegExp(
```

Comment:
Is there a reason you are using `RegExp` to generate the regex rather than setting it yourself as a string? EG: `/(\/static\/js\/(main|vendor|${service})-\w+\.\w+\.js)/g`?

I only ask because I think it's easier to understand the regex as an explicit string rather than using `RegExp` to generate it during to the lack of double `//`

Reply:
Ignore me, having looked further this is the nicest way to programmatically add the service variable to the regex.